_data/navigation.yaml

```diff
@@ -36,6 +36,9 @@
     - name: MeshMates
       link: /community/meshmates
       new_window: false
+    - name: Members
```

Comment:
I'd like to leave this link outside of the main menu. Arguably, the MeshMates link could be removed as well. We can wait to do that for the layer5ng site, though.

Reply:
Okay,  I will see if it can be inserted in the community page somewhere, or not.